tests/FSharp.Compiler.ComponentTests/Conformance/UnionTypes/LowercaseWhenRequireQualifiedAccess.fsx

```diff
@@ -0,0 +1,44 @@
+﻿// #Conformance #TypesAndModules #Unions 
+// Allow lower-case DU cases when [<RequireQualifiedAccess>] is specified
+//<Expects status="success"></Expects>
+#light
+
+[<RequireQualifiedAccess>]
+type DU1 =
+    | a
```

Comment:
Please add a test case with a dodgy lower case name that is not a valid type name for .NET, e.g.
```fsharp
type DU =
    | ``not.allowed``
```

Reply:
Added more testing. Thanks for pointing this out .

Reply:
Yeah, let's put it under preview version.